packages/web/src/public-site/components/NavBanner.tsx

```diff
@@ -57,6 +57,16 @@ type NavBannerProps = {
   setRenderPublicSite: (shouldRender: boolean) => void
 }
 
+/**
+ * A button element with no styling, purely for using the correct html role
+ */
+const UnstyledButton = (props: HTMLAttributes<HTMLButtonElement>) => (
```

Comment:
Should I put this in somewhere common?

Reply:
yeah lets do that cause we can use it for "forgot-password" link in sign-up where in reality its a button, and we'd want it unstyled like this. if you'd like, just in src/components is good